docs/user-guide/content-image-toolbar.md

```diff
@@ -1,16 +1,16 @@
 # Image Content Toolbar
 **********************
 
-As soon as an image content is added, the Content Image Toolbar appears on top of the image: 
+As soon as an image content is added, the Image Content Toolbar appears on top of the image: 
 
-<img src="../img/content-image-toolbar/c-img-toolbar.jpg" class="ms-docimage"/>
+<img src="../img/content-image-toolbar/image-toolbar.jpg" class="ms-docimage"/>
 
 Through this toolbar, the story editor is able to perform the following operation:
 
 * **Change media source** <img src="../img/button/change-media2.jpg" class="ms-docbutton"/> accessing the [Media Editor](media-editor-window.md)
 
 * **Change size** <img src="../img/button/change-size2.jpg" class="ms-docbutton"/> choosing between *Small*, *Medium*, *Large*
 
-* **Align content** <img src="../img/button/align2.jpg" class="ms-docbutton"/> on the *Left*, *Center* or *Right*
+* **Hide caption** button <img src="../img/button/hide-caption.jpg" class="ms-docbutton"/> to show/hide the description under the image. (It is available if the description is added to the image on the [Media Editor](media-editor-window.md))
```

Comment:
* **Hide caption** button <img src="../img/button/hide-caption.jpg" class="ms-docbutton"/> to show/hide the description under the image: this button is present only if a description has been provided for the image through the [Media Editor](media-editor-window.md)